_layouts/default.html

```diff
@@ -44,7 +47,7 @@
 
     <footer>
         <div class="divider-line"></div>
-        <div class="row certs shifted-down">
+        <div class="certs d-flex flex-column flex-lg-row row-cols-lg-4">
```

Comment:
I actually don't think any of these flex classes are necessary. If this was a `div` with a `row` class, with four child `div`s with a `col-md-3`, then the columns would respond the way we want them to: 100% width in small widths, and then 25% on wider screens.

Reply:
Save this for a future refactor. Resolving for now.